interfaces.go

```diff
@@ -0,0 +1,21 @@
+package vfsgen
+
+// GzipByter is implemented by compressed files for
+// efficient direct access to the internal compressed bytes.
+type GzipByter interface {
+	// GzipBytes returns gzip compressed contents of the file.
+	GzipBytes() []byte
+}
+
+// TODO: Choose one from below.
+
+// NotWorthGzipCompressing is implemented by files that were determined
+// not to be worth gzip compressing (the file size did not decrease as a result).
+type NotWorthGzipCompressing interface {
+	NotWorthGzipCompressing()
+}
+
+// NotWorthGzipCompressing indicates the file is not worth gzip compressing.
+type NotWorthGzipCompressing_ interface {
```

Comment:
At some point I wanted to consolidate the `GzipByter` and `NotWorthGzipCompressing` interfaces into one that provides either compressed bytes, or lets you know it wasn't worth compressing, but I couldn't find a good common interface and decided to leave it as 2 separate ones.